components/KiboOrderSummary.vue

```diff
@@ -1,106 +1,68 @@
 <template>
   <div class="sf-order-summary">
-    <slot name="heading" v-bind="{ orderTitle, orderTitleLevel }">
-      <SfHeading
-        :title="orderTitle"
-        :level="orderTitleLevel"
-        class="sf-heading--left sf-heading--no-underline sf-order-summary__heading"
-      />
-    </slot>
-    <div class="highlighted highlighted--total">
-      <slot
-        name="summary"
-        v-bind="{
-          totalItems,
-          subtotal,
-          total,
-          shippingMethod,
-          propertiesNames,
-        }"
-      >
+    <div class="sf-order-summary__heading">{{ orderTitle }}</div>
+    <div class="sf-order-summary__content">
+      <div class="props">
         <SfProperty
           :name="propertiesNames[0]"
```

Comment:
Do we need to make the name dynamic when the values are static ?

Reply:
Just have followed SfOrderSummary patten, but yes we can use it directly as well

Reply:
would it make more sense for this to be an object for labels? instead of an array?

Reply:
Resolved. Used Object instead of array